build-support/bin/BUILD

```diff
@@ -1,66 +1,172 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
This PR adds _all_ `build-support/bin` entries as targets. This is not strictly necessary to get this test working, but is a good practice that we should get in the habit of doing. Now tests will be able to safely depend on these files by marking the dependency in `BUILD`.

Reply:
Hm. I don't think that we should be doing "target per file" in here. The maintenance of that mode is unlikely to be worth the cost.

I expect that it would be reasonable to collect all of the non-python files into one `files` target, as there is less benefit to having fine-grained target for loose files than there is for source code.

Reply:
Sounds good! 

In general I'm a big fan of per-target because it makes it easier to quickly run the tests you care about, is easier to add `type_checked` tag, and makes V2 use less files in the PEX. But only the last reason applies in this instance, and I agree that it's not worth the cost of complexity.